Fix #3411 fix button classes on collection editor

zvekovius opened the pull request:
Fix #3411 for button classes on the collection editor. 
I got 401's trying to access the collection editor on my dev server, but the changes seemed trivial enough.

seanlip commented on core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html:
```diff
@@ -5,6 +5,6 @@
-  <input ng-model="newExplorationId" type="text" placeholder="New exploration ID">
-  <button type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
+  <input class="form-inline" ng-model="newExplorationId" type="text" placeholder="New exploration ID">
+  <button class="btn btn-success" type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
```
Can we use "btn btn-default" for this? I think we shouldn't have two green buttons; it diverts attention.

zvekovius replied:
Sure. I can make the change tonight.

seanlip commented:
Passing to @shaz13 for review and testing.

shaz13 commented on core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html:
```diff
@@ -5,6 +5,6 @@
 <form ng-submit="addExploration()">
-  <input ng-model="newExplorationId" type="text" placeholder="New exploration ID">
-  <button type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
+  <input class="form-inline" ng-model="newExplorationId" type="text" placeholder="New exploration ID">
```
This has no effect. And, your branch seems to have the collection editor bug (PR #3410). To access it you have to edit the files locally from the mentioned PR or update if it's merged. 

> I got 401's trying to access the collection editor on my dev server, but the changes seemed trivial enough.

From the development perspective, it is very dangerous to push the code blindly. Please verify the changes locally before making a PR

zvekovius replied:
I'll add those changes to my branch and try again. Looks like it still hasn't merged.

seanlip replied:
@zvekovius -- my apologies for the delay, it's merged now. Please merge develop into your branch and let me know if it still doesn't work.

Also, note that you'll need to go to /admin, go to the config tab, and add your username to the list of users there in order to access the collection editor, otherwise you'll still get a 401.

Thanks!

zvekovius replied:
@seanlip No worries, I just threw the changes in. I just commented out the auth requirements and got to the page :) 
Got the changes fixed up. I also changed to use btn-default. 
Attached SS. 
Look good, or should we go back to btn-success?
![image](https://cloud.githubusercontent.com/assets/17284745/25979330/6d72d8b8-368d-11e7-8634-53b2f282981a.png)

seanlip replied:
Thanks, @zvekovius! I'm fine with this -- @shaz13, what do you think?

shaz13 replied:
`btn-default` looks much better. Go for it

zvekovius replied:
@seanlip  - I'll work on getting the new commits to my branch tonight. Was running into some issues last night. 
Just noticed this as well. It says "New Exploration ID" when we are adding an existing exploration. Should we change it to just "Exploration ID"?

shaz13 replied:
I will defer the change to @seanlip. 

Note:  I can see "New exploration ID" (placeholder) comes from its `ng-model`. And this naming continues in respective JS files. @zvekovius Are you suggesting changing just the placeholder value ? Or renaming the ng-model reference itself (In that case it could be filed under separate issue) 
```
<input ng-model="newExplorationId" type="text" placeholder="New exploration ID">
```

seanlip replied:
You know, that is a good point @zvekovius (I assume you're referring to what is visibile by the user). I'd lean towards "Existing exploration ID" to match the button text. What do you think?

I think keeping the internal variable as newExplorationId is fine.

shaz13 replied:
Agree

zvekovius replied:
Yeah, I was talking the placeholder value form the input field. 
I'll change it to "Existing Exploration ID" with the next commit as well.

zvekovius commented:
@seanlip @shaz13 
Sorry for the delay. Had some fun with my dev box last few days. Hard drives are great. 
New commits on branch. Changed classes on the 2nd form, input box, and button. Also changed the place holder text for the input box.
Should be good to go.

seanlip commented on core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html:
```diff
@@ -1,10 +1,10 @@
 <h2>Add Exploration</h2>
 <form class="form-inline" ng-submit="createNewExploration()">
-  <input class="form-control" ng-model="newExplorationTitle" type="text" placeholder="New exploration title">
+  <input class="form-control" ng-model="newExplorationTitle" type="text" placeholder="New Exploration title">
```
Why capitalize "Exploration" here but not below? (Probably want to uncapitalize this one.)

seanlip commented:
One minor comment. Otherwise LGTM!

zvekovius commented:
@seanlip  - Done.

shaz13 commented:
LGTM! Thanks @zvekovius
